### ISSUE TITLE
`ALL` selector inconsistency between `ruff` and `ty`

### ISSUE THREAD

**Issue by Andrej730** [astral-sh/ty#4079]
### Summary

Kind of minor thing, but I'll report it...

In ruff, a way to select all rules is `ALL`, `all` won't work.
```sh
ruff check --select ALL
# Cause: Unknown rule selector `all` in `select` from the CLI
ruff check --select all
```

While in ty...
```python
ty check --error all
# warning[unknown-rule]: Unknown rule `ALL`
ty check --error ALL
```

I guess in ruff it comes from all rule codes defined in uppercase and `ty` not having rule codes at all.
A minor discrepancy, but something I always need to remember when switching between the tools.

### Version

0.0.61
